src/amo/components/FeedbackForm/styles.scss

```diff
@@ -71,8 +71,12 @@ input[type='checkbox'] + .FeedbackForm--label {
 .FeedbackForm--help {
   color: $grey-50;
   display: block;
-  font-size: $font-size-xs;
-  margin-top: 4px;
+  margin-top: 0;
+  max-width: 700px;
```

Comment:
I don't think we should be breaking this text over several lines like that ?

Reply:
That's coming from UX because the text is too long otherwise (and still multi-line).

Reply:
mmm I think it looks quite weird given the fields themselves (message, but also the location select) are full width...

Reply:
ok, I revert for now..